api/HTMLInputElement.json

```diff
@@ -95,33 +94,32 @@
           }
         }
       },
-      "autofocus": {
+      "align": {
```

Comment:
`align` is listed in https://html.spec.whatwg.org/multipage/obsolete.html#HTMLInputElement-partial and should be deprecated. (But I can't make suggestions in that part of the diff.)